mise-en-page/accueil-institution-directives.html

```diff
@@ -153,7 +150,7 @@ <h1 property="name" id="wb-cont">[Nom de l'institution]</h1>
 					<div class="row guidance-details">
 						<div class="col-md-8">
 							<details>
-								<summary><h2 class="h4">Directives&nbsp;:</h2> Titre, image et bouton vers une super-tâche <i class="fas fa-info-circle"></i></summary>
```

Comment:
Didn't we want to remove the section in the guidance pieces instead?

Reply:
A bunch of guidance pieces have h3s in them, which like... technically are headings? 

I can take out the `<sections>`

Reply:
I was thinking more about removing the `h3`s to replaced them with `p.h3` and removing the sections. That way the guidances are not part of the document outline. Unless we want the guidances to be included than removing the sections should be okay.